README.md

```diff
@@ -46,30 +46,122 @@ Simply run the following:
 
 ```
 $ go install ./cmd/cloud
+alias cloud='$HOME/go/bin/cloud'
 ```
 
-### Running
 
+### Pre-requisites
```

Comment:
This section is nice to have, but it seems like we should try to combine it with the `Environment Setup` section above because they contain the same information. Actually, it looks like there is now a discrepancy on Helm version recommendation with this change so we should definitely update/combine these sections.

Reply:
you are right Gabe, I have completely missed that. I am updating the setup section.